packages/gasket-plugin-webpack/docs/webpack.md

```diff
@@ -62,13 +62,13 @@ module.exports = {
   hooks: {
     webpack(gasket, webpackConfig) {
       // construct and return some webpack config partial
-      return {
+      return webpackMerge(webpackConfig, {
```

Comment:
This isn't correct, though. `@gasket/webpack-plugin` already uses `webpackMerge` on the config partials emitted for the `webpack` lifecycle:

https://github.com/godaddy/gasket/blob/master/packages/gasket-plugin-webpack/index.js#L40

Reply:
fixed it 👍🏽